orchestrator/settings.py

```diff
@@ -77,6 +77,7 @@ class AppSettings(BaseSettings):
     DEFAULT_CUSTOMER_FULLNAME: str = "Default::Orchestrator-Core Customer"
     DEFAULT_CUSTOMER_SHORTCODE: str = "default-cust"
     DEFAULT_CUSTOMER_IDENTIFIER: str = "59289a57-70fb-4ff5-9c93-10fe67b12434"
+    DAYS_TO_KEEP_OLD_TASKS: int = 3
```

Comment:
Ik vind iets als `TASK_LOG_RETENTION_DAYS` oid mooier voor een config naam

Reply:
Good suggestion, changed it.